assets/demos/README.md

```diff
@@ -0,0 +1,97 @@
+# Benchpress Demo Application
+
+* Clone AspNetCore.Docs repository
+
+```powershell
+git clone https://github.com/dotnet/AspNetCore.Docs.git
+```
+
+* Create a new resource group with a unique name
+
+```powershell
+$suffix = (Get-Random).ToString("x8")
+$location = "westus2"
+
+az group create --name "benchpress-rg-${suffix}" --location "${location}"
+```
+
+* Deploy the demo application infrastructure
```

Comment:
Please provide links to the documentation for each resource type to aid users not familiar with said resources.

Reply:
good feedback, thanks 👍